test/unit/org/apache/cassandra/utils/ASTGenerators.java

```diff
@@ -391,6 +393,18 @@ public MutationGenBuilder(TableMetadata metadata)
                 columnExpressions.put(symbol, new ExpressionBuilder(symbol.type()));
         }
 
+        public MutationGenBuilder withAllowPartitionOnlyUpdate(boolean value)
```

Comment:
Unused?

Reply:
unused but there to give an opt-out switch if the user doesn't want `IN` clause support for partitions